planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/src/node.cpp

```diff
@@ -27,45 +26,19 @@
 #include <obstacle_stop_planner/node.hpp>
 #include <vector>
 #define EIGEN_MPL2_ONLY
-#include <Eigen/Core>
-#include <Eigen/Geometry>
+#include <eigen3/Eigen/Core>
+#include <eigen3/Eigen/Geometry>
 namespace
 {
-template <class T>
-T getParam(const ros::NodeHandle & nh, const std::string & key, const T & default_value)
```

Comment:
Ahh, I see these functions seem to left over from a common code source?

Reply:
they were used to handle parameters so we don't need them anymore